core/codegen/src/utils/parser_ext.rs

```diff
@@ -14,6 +15,10 @@ pub trait ParserExt<'a> {
 
     // Like `parse_ident` but also looks for an `ident` in a `Pat`.
     fn parse_ident_inc_pat(&mut self) -> PResult<'a, Ident>;
+
+    // Duplicates previously removed method in libsyntax
```

Comment:
Should this link back to the rust repository more concretely? I wasn't sure if I should refer to a PR, commit, URL, or just leave a generic note like this.

Reply:
Let's link to the PR that removed the method. Something like, "See https://github.com/rust-lang/rust/foo/bar/baz for more details.".